code/src/java/pcgen/system/BatchExporter.java

```diff
@@ -471,6 +472,16 @@ public boolean accept(File aFile, String aString)
 		});
 	}
 
+	public static void printToXmlStream(CharacterFacade character, OutputStream outputStream)
```

Comment:
I actually have a bunch of javadoc changes in the commit "Deleted FOPHandler in preference to FopTask" but I rolled it back because I wanted to check that commit in a separate pull request.